mitodl/ocw-www
# add instructors

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-www/issues/126

#### What's this PR do?
In https://github.com/mitodl/ocw-studio/pull/518, we started publishing instructors imported from legacy OCW content imported from `ocw-to-hugo`.  Since we are using this repository as a content source for building `ocw-www` until we are ready to move to purely `ocw-studio` driven content, it will need to be updated with these instructors as a way to test and develop the static JSON API that will be built by Hugo and accessed by course sites to get instructor information.

#### How should this be manually tested?
 - Clone https://github.com/mitodl/ocw-hugo-themes on the `cg/instructors-static-api` branch
 - Clone https://github.com/mitodl/ocw-hugo-projects
 - Start the webpack dev server with `npm run start:site:webpack`
 - In your `ocw-www` folder on this PR branch, run `hugo server --config /path/to/ocw-hugo-projects/ocw-www/config.yaml --themesDir /path/to/ocw-hugo-themes`
 - Pick an instructor in the source `ocw-www` data and visit your local dev site verifying that the instructor exists and data is returned, for example: http://localhost:1313/instructors/0a90e4c5-5f99-646e-cbfc-402250d8aba5/index.json
